zenip-42203.md

```diff
@@ -0,0 +1,31 @@
+# ZenIP-42203
+
+    ZenIP: 42203
+    Title: HorizenDAO
+    Owners: Horizen Community Council, Erica Hamilton erica@horizen.io, Dean Steinbeck dean@horizenlabs.io, Gustavo Fialho gustavo@zensystem.io 
+    Status: Draft
+    Type: Consensus
```

Comment:
This ZenIP doesn't fall within the `Consensus` type, the appropriate type would be `Process`.

Please review the ZenIP process specification [zenip-42000.md#zenip-types](https://github.com/HorizenOfficial/ZenIPs/blob/bf607b30ce62d9dbae46ec02861e8255e59bea8a/zenip-42000.md#zenip-types) and assign the correct type. Please leave the ZenIP number/filename blank for later assignment by a ZenIP editor.

Reply:
Updated